spyder/plugins/application/container.py

```diff
@@ -297,22 +315,38 @@ def _check_updates_ready(self):
                         "<code>conda install spyder={}</code><br><br>"
                     ).format(latest_release)
                 else:
-                    content = _(
-                        "Click <a href=\"{}\">this link</a> to "
-                        "download it.<br><br>"
-                    ).format(url_r)
+                    if is_pynsist() or running_in_mac_app():
+                        box.setStandardButtons(QMessageBox.Yes |
+                                               QMessageBox.No)
+                        content = _(
+                            "Would you like to automatically download and"
+                            " install the latest version of Spyder?<br><br>"
+                        )
+                    else:
+                        content = _(
+                            "Click <a href=\"{}\">this link</a> to "
+                            "download it.<br><br>"
+                        ).format(url_r)
```

Comment:
I think this is not needed anymore as well as the `url_r` variable definition above, so:
```suggestion

```

And also lines 266-273 should be deleted